src/o365/spfx/commands/project/project-upgrade/model/ReportData.ts

```diff
@@ -1,4 +1,5 @@
-import { Hash, Dictionary } from "./project-upgrade/";
+import { Hash } from "../Hash";
```

Comment:
ReportData is not a part of an SPFx project, but rather our own class, right? As such, we shouldn't put it in the `model` folder that is meant for classes that represent SPFx project structure.

Reply:
I followed the same approach as project upgrade here

Reply:
rolled back the change now that I'm not using report data anymore